nullaway/src/main/java/com/uber/nullaway/generics/GenericsChecks.java

```diff
@@ -768,6 +888,40 @@ public static Nullness getGenericParameterNullnessAtInvocation(
       MethodInvocationTree tree,
       VisitorState state,
       Config config) {
+    // If generic method invocation
```

Comment:
Similar comment to the one above about a generic instance method of a generic type

Reply:
Do you mean the Javadoc comment? Would you want me to erase this?

Reply:
I just meant that this code may not work in cases like https://github.com/uber/NullAway/pull/1053/files#r1805634821.  Nothing to do here for now I think, we can look in a follow up.